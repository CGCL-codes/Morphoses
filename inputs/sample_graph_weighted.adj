WeightedAdjacencyGraph
22
19
0
0
1
3
4
4
5
7
8
10
10
11
12
13
14
15
15
17
17
18
19
19
2
3
4
4
4
7
5
8
5
2
12
20
19
17
21
14
14
20
12
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
