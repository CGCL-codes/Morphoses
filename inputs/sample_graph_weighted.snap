1	2	1
2	3	1
2	4	1
3	4	1
5	4	1
6	7	1
6	5	1
7	8	1
8	5	1
8	2	1
10	12	1
11	20	1
12	19	1
19	12	1
18	20	1
14	21	1
13	17	1
16	14	1
16	14	1
17	16	1